viadot/sources/mindful.py

```diff
@@ -222,4 +218,4 @@ def response_to_file(
                 "File extension is not available, please choose file_extension: 'parquet' or 'csv' (def.) at Mindful instance."
             )
 
-        return complete_file_name
+        return absolute_path
```

Comment:
You are returning absolute_path. In function definition is `Response` and `request object with the response from the Mindful API.` in docstring. Is it correct?

Reply:
It is not the absolute path, but the relative. I have changed the name to not be confusing.